scripts/utils/sync-ws-auth-js.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# A command that modifies the package.json of all workspaces that depdend on @okta/okta-auth-js to use a consistent version
+
+workspaces=$(yarn -s workspaces info | jq 'map(..|objects|select(.location).location)[1:] | @sh' | tr -d \'\")
```

Comment:
possible to turn this one into a bacon task from okta-react side, so we can keep versions synced for each PR

Reply:
or is there a lint rule of that for monorepo?

Reply:
This script is only ran when running auth-js ---> okta-react downstream builds. This script ensures all yarn workspaces in the okta-react repo are using the upstream auth-js version, rather than installing their own copy from npm/yarn.

This script is only ran in `setup.sh` in the ` if AUTHJS_VERSION` (aka if downstream build) block